ext/project_data.yaml

```diff
@@ -1,51 +0,0 @@
----
-project: 'razor-server'
-author: 'Puppet Labs'
-email: 'info@puppetlabs.com'
-homepage: 'http://puppetlabs.com/puppet/puppet-enterprise'
-summary: 'Razor is an advanced provisioning application'
-description: |
-  Razor is an advanced provisioning application which can deploy both bare-metal
-  and virtual systems. It's aimed at solving the problem of how to bring new
-  metal into a state where your existing DevOps/configuration management
-  workflows can take it over.
-
-  Newly added machines in a Razor deployment will PXE-boot from a special Razor
-  Microkernel image, then check in, provide Razor with inventory information, and
-  wait for further instructions. Razor will consult user-created policy rules to
-  choose which preconfigured model to apply to a new node, which will begin to
-  follow the model's directions, giving feedback to Razor as it completes various
-  steps. Models can include steps for handoff to a DevOps system such as Puppet
-  or to any other system capable of controlling the node (such as a vCenter
-  server taking possession of ESX systems).
-# automatically burn our version into the generated source
-version_file: 'lib/razor/version.rb'
-update_version_file: true
-# files and gem_files are space separated lists (or arrays)
-files:
```

Comment:
Does this list actually exist somewhere in razor-vanagon, or should it?

Reply:
I believe this list is only needed for old-fashioned building with the packaging repo